web/src/app/chat/documentSidebar/DocumentSidebar.tsx

```diff
@@ -132,15 +133,34 @@ export function DocumentSidebar({
 
       <div className="text-sm mb-4 border-t border-border pt-4 overflow-y-hidden flex flex-col">
         <div className="flex border-b border-border px-3">
-          <div>
+          <div className="flex">
             <SectionHeader name="Selected Documents" icon={FiFileText} />
+
+            {tokenLimitReached && (
+              <div className="ml-2 my-auto">
+                <div className="mb-2">
+                  <HoverPopup
+                    mainContent={
+                      <FiAlertTriangle
+                        className="text-alert my-auto"
+                        size="16"
+                      />
+                    }
+                    popupContent={
+                      <Text className="w-40">
+                        Over LLM context length by:{" "}
```

Comment:
How do you feel about:
Over LLM context limit by: x tokens
Truncating <document name>

Reply:
Also maybe the warning symbol can be a bit larger? I'm worried people won't see it :|